CONTRIBUTING.md

```diff
@@ -6,6 +6,7 @@ Ruby for Good [code of conduct](https://github.com/rubyforgood/code-of-conduct).
 **First:** if you're unsure or afraid of *anything*, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions, and don't want a wall of rules to get in the way of that.
 
 ## Onboarding the App
+You can find detailed instructions on how to seup the app locally in [installation.md](/installation.md).
```

Comment:
Should we even have the rest of this section here if we're going to keep installation.md as the source of truth?

Reply:
I think there's a valid argument for removing it. But installation doesn't reference the bin/setup script at all.

Is `installation.md` ever used by folks who are just hosting human essentials? I wondered if there were different purposes between these docs.

Reply:
Good question. I'm not entirely sure... I'm happy to punt on that for now until Edwin or someone who knows more can answer. In the meantime I'm happy to merge this all in as it provides more up to date info for contributors.